### DISCUSSION TITLE
Is it possible to install SpotX on my current old version of spotify?

### DISCUSSION THREAD

**Discussion by toblerone567** [SpotX-Official/SpotX#158] [Question] [answered]
I've been using version 1.1.46.916.g416cacf1 of Spotify for almost a year now because of the actually decent UI,  and I'd like to install SpotX and not have it overwrite my current installation. Is that possible? Thanks in advance

**Comment by NeuroCPP** [comment]
1.Please specify if are using any other third party patch (means if you're client is modeified or not)
2.What are you using Spotx for?
 if for blocking those pesky ads then
it might be possible possible with [bts](https://github.com/mrpond/BlockTheSpot) patch
but it might not work with Spotx patch(starting from v1.5)(as it modifies xpui.js)(am I right ?@amd64fox)
so you will need to add -bts parameter.(read more about parameters [here](https://github.com/amd64fox/SpotX/discussions/60)
for easy installation use this.
[Install_Auto.txt](https://github.com/amd64fox/SpotX/files/9632027/Install_Auto.txt)
**change the file extention ".txt" to ".bat"**
for example to not install recommended version :
![image](https://user-images.githubusercontent.com/85978957/191920400-97b611aa-ee16-427d-91c0-2349921a219d.png)

**Reply by amd64fox** [reply]
SpotX has recently been blocking all ads on its own, it used to work in symbiosis with the `bts` patch, now it is not necessary, but I left the option to use the `bts` patch since it was recently updated and it also works.

**Comment by amd64fox** [comment]
do you have a modified version `1.1.46.916.g416cacf1` or the official one?

**Comment by toblerone567** [comment, marked as answer]
1: It's the official version, I just blocked updating
2: I'm just using it for the ad blocking

**Reply by amd64fox** [reply]
Two Spotify cannot be installed at the same time, so you can uninstall your old Spotfiy and run a normal install of SpotX, if you don't like SpotX, then you can revert to your old version of Spotify with [this](https://github.com/amd64fox/Rollback-Spotify).
